setup.py

```diff
@@ -2,7 +2,7 @@
 
 
 setup(name='pythonequipmentdrivers',
-      version='1.10.0',
+      version='1.11.0',
```

Comment:
1.10.1

(Major Update).(Minor Update).(Patch/Bugfix)

Since your just adding a few commands to an existing device I would call this a patch.

Reply:
changed version number.
also fixed typo in set_channel_findscale docstring